package.json

```diff
@@ -8,7 +8,7 @@
     "webpack-defaults": "webpack-defaults",
     "start": "npm run build -- -w",
     "appveyor:test": "npm run test",
-    "build": "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
```

Comment:
ah that's fine, will revert it later. wasn't sure if it was needed as i assumed tests would be in jest's usual tests directory.